torchgeo/trainers/byol.py

```diff
@@ -368,11 +361,13 @@ def __init__(self, **kwargs: Any) -> None:
             ValueError: if kwargs arguments are invalid
         """
         super().__init__()
-        self.save_hyperparameters()  # creates `self.hparams` from kwargs
+
+        # Creates `self.hparams` from kwargs
+        self.save_hyperparameters()  # type: ignore[operator]
```

Comment:
For some reason mypy thinks this is a `Tensor`, not a `Callable`. We could cast it but I'm not sure exactly what kind of `Callable` this is. I'm going to chalk this up to "pytorch-lightning is super hacky" and leave it for future work to figure out why this doesn't work.